include/ccf/kv/version_v.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+namespace kv
```

Comment:
That doesn't seem like it should be public?

Reply:
Good spot! The only bit of these we need was already in `version.h`, so this has returned to the implementation.